projects/storefrontlib/src/shared/utils/validators/custom-form-validators.spec.ts

```diff
@@ -167,8 +173,8 @@ describe('FormValidationService', () => {
   });
 
   describe('Star rating validator', () => {
-    const invalidValues = [null, 'a', 0, 1000];
-    const validValues = [1, 2, 3, 4, 5];
+    const invalidValues = [null, 'a', 0, VAL_1000];
+    const validValues = [1, VAL_2, VAL_3, VAL_4, VAL_5];
```

Comment:
Rule exclude 0 & 1 because they're very common (independly) in code. But It seems so ugly if we mix constants and naked numbers in arrays like here.

BTW Is that rule failing while we we define array with not defined numbers? Maybe there is possibility to turn off that

Reply:
If I define like this: const validValues:String[] = ['1', '2', '3', '4', '5'];
then there is no such error.